keyboards/keebio/quefrency/keymaps/bfiedler/keymap.c

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2021 Ben Fiedler
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// https://beta.docs.qmk.fm/using-qmk/software-features/tap_hold
+#define TAPPING_TERM 120
+#define IGNORE_MOD_TAP_INTERRUPT
+#define PERMISSIVE_HOLD
+
+// This does not disable RGB entirely, but it disables it by default.
+#undef RGBLIGHT_ENABLE
```

Comment:
If you want rgblight disabled, putting it into the rules.mk is the proper way.  Otherwise, this only disables the check locally, to this file.

Reply:
Ah, thanks. I originally wanted to be able to enable them post-startup, but I've gone with the permanent disable anyway.